.github/workflows/ci_emmocheck.yml

```
@@ -26,4 +26,4 @@ jobs:
 
     - name: Check EMMO
       run: |
-        emmocheck --local --verbose --check-imported crystallography.owl
+        #emmocheck --url-from-catalog --verbose crystallography.owl
```

```suggestion
        emmocheck --url-from-catalog --verbose crystallography.owl
```
Shouldn't this be fine to run now, or?

---

See my [other comment below](https://github.com/emmo-repo/domain-crystallography/pull/5#issuecomment-726146575).